__tests__/app.test.js

```diff
@@ -342,3 +342,95 @@ describe("GET /api/users", () => {
       });
   });
 });
+//
+describe("GET /api/reviews(queries)", () => {
+  it("200: should respond with the reviews by the category value specified in the query", () => {
+    const category = "euro game";
+    return request(app)
+      .get(`/api/reviews?category=${category}`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(typeof body).toBe("object");
+        const { reviews } = body;
+        reviews.forEach((review) => {
+          expect(review).toHaveProperty("category", category);
+        });
+      });
+  });
+  //
+  it("200: should respond with the reviews sorted by the value specified in the query default to date", () => {
+    const sort_by = "votes";
+    return request(app)
+      .get(`/api/reviews?sort_by=${sort_by}`)
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toBeSortedBy(`${sort_by}`, {
+          descending: true,
+        });
+      });
+  });
+  //
+  it("200: should respond with the reviews sorted in asc order and the default should be desc", () => {
+    const order = "asc";
+    return request(app)
+      .get(`/api/reviews?order=${order}`)
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toBeSortedBy(`created_at`, {
+          descending: false,
+        });
+      });
+  });
+  //
+  it("200: should respond with the reviews by category, sorted by value and in order", () => {
+    const category = "social deduction";
+    const sort_by = "votes";
+    const order = "asc";
+    return request(app)
+      .get(
+        `/api/reviews?category=${category}&sort_by=${sort_by}&order=${order}`
+      )
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        reviews.forEach((review) => {
+          expect(review).toHaveProperty("category", category);
+        });
+        expect(reviews).toBeSortedBy(`${sort_by}`, {
+          descending: false,
+        });
```

Comment:
This would be a good place to add one more 200 test that responds with an empty array if the category is valid but no articles in it 🤔